src/components/NFTClaim/OptionList.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <Dropdown
+    :content-classes="[
+      'absolute',
+      'right-0',
+      'bottom-full',
+      'mt-[8px]',
+      'z-10',
+    ]"
+  >
+    <template #trigger="{ toggle }">
+      <ButtonV2
+        class="w-full underline text-medium-gray"
+        preset="plain"
+        :is-disabled="!isContentViewable"
+        @click="toggle"
+      >
+        <template #default>{{ $t('nft_claim_claimed_download') }}</template>
+      </ButtonV2>
+    </template>
+    <MenuList>
+      <ul>
+        <li
+          v-for="(contentUrl, index) in normalizedContentURLs"
+          :key="contentUrl"
+        >
+          <ButtonV2
+            :href="parseNFTMetadataURL(contentUrl)"
+            preset="plain"
+            :download="getDownloadFilenameFromURL(contentUrl)"
+            @click="e => handleClickViewContentURL(e, contentUrl, index)"
+            >{{
+              getFilenameFromURL(contentUrl) ||
+                getContentUrlButtonText(contentUrl)
+            }}&nbsp;<IconLinkExternal
+          /></ButtonV2>
```

Comment:
```suggestion
          <ButtonV2
            :href="parseNFTMetadataURL(contentUrl)"
            preset="plain"
            :download="getDownloadFilenameFromURL(contentUrl)"
            @click="e => handleClickViewContentURL(e, contentUrl, index)"
          >
            {{
              getFilenameFromURL(contentUrl) ||
                getContentUrlButtonText(contentUrl)
            }}&nbsp;<IconLinkExternal />
          </ButtonV2>
```

Reply:
[🎨 Improve formatting in OptionList](https://github.com/likecoin/liker-land/pull/1588/commits/65fac3b0a4c1b3c57154bed91f9fd728a471d17c)